cfme/tests/configure/test_access_control.py

```diff
@@ -1336,17 +1339,17 @@ def test_tenant_quota_input_validate(appliance):
 
 def test_delete_default_tenant(appliance):
     """
-    Steps:
-    1. Login as an 'Administrator' user
-    2. Navigate to configuration > access control > tenants
-    3. Select default tenant('My Company') from tenants table
-    4. Delete using 'configuration > Delete selected items'
-    5. Check whether default tenant is deleted or not
-
     Polarion:
         assignee: ghubale
         casecomponent: Infra
         initialEstimate: 1/20h
+        tags: quota
+        testSteps:
+            1. Login as an 'Administrator' user
+            2. Navigate to configuration > access control > tenants
+            3. Select default tenant('My Company') from tenants table
+            4. Delete using 'configuration > Delete selected items'
+            5. Check whether default tenant is deleted or not
```

Comment:
**Required** Please break this up between `testSteps` and `expectedResults`

Reply:
As per me it should contributors privilege else we need to add large change in integration test. For every step we need expectedResults.

Reply:
I talked to Mike about this, for automated test we do not need to separate it so strictly. So this can be **Optional**